tests/Hedgehog.Tests/tests-js/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Hedgehog.Fable tests</title>
```

Comment:
This name should be updated. Also this appears to be a duplicate of `tests/Hedgehog.Tests/index.html`, are both versions required?

Reply:
Renamed to Hedgehog.Tests.
Oh, tests-js should not be commited to git 😅. Removed and gitignored it.